### PR TITLE
Refactored readme and ammonite quickstart

### PR DESCRIPTION
**Pull request by ngortheone**
Some documentation improvements:
* added Travis-CI badge
* added Maven Central badge
* Quick start section moved closer to the beginning
* Added [Ammonite-REPL](http://ammonite.io/#Ammonite-REPL) script for quick experimentations
* More involved examples moved to separate file `docs/Examples.md`

You can quickly preview how readme looks in [my fork](https://github.com/ngortheone/skuber/tree/quickstart)

### CODE REVIEW

**Comment by doriordan** [comment]
Thanks for this @ngortheone, very useful additions in general. I haven't used Ammonite myself, so I'll take a closer look at that but it looks like a nice enhancement.
The addition of Travis build status and Maven Central badge have been on my todo list so it will be good to have them now.
I personally like having at least one or two short examples in the main README - it is something that gives complete newcomers  to skuber a quick orientation in terms of how it is used. 
Hence I'd prefer to move some of the content from `Examples.md` into the main README (probably as an `Examples` section just before the quickstart section). I think the `list` example  is simple and short enough for a front page, the longer deployment example can remain in `Examples.md` and be referenced from the `Examples` section in the front page.

**Comment by ngortheone** [comment]
Sure @doriordan, I will put list example back to the front page then

**Comment by ngortheone** [comment]
@doriordan as per your feedback I have returned code example to the main page. I put it as one of the items in quick start section - this way the doc looks more composed to me. I can surely put it into a separate `## Examples` section if you want.

File: README.md [hunk 1]
```diff
@@ -7,125 +11,131 @@ Skuber is a Scala client library for [Kubernetes](http://kubernetes.io). It prov
...
 
-```sbt
-libraryDependencies += "io.skuber" %% "skuber" % "2.0.9"    
-```
+  ```
+  import $ivy.`io.skuber::skuber:2.0.9`, skuber._, skuber.json.format._
+  ```
 
-Meanwhile users of skuber v1 can continue to use the latest (and possibly final, with exception of important fixes) v1.x release, which is available only on Scala 2.11:
+### Copy-paste ready snippet
```
Comment (doriordan) [inline README.md]: As a sub-heading can we retitle this to `Example`, and note that t can be copied/pasted in the first line of the following paragraph?

File: README.md [hunk 2]
```diff
@@ -7,125 +11,131 @@ Skuber is a Scala client library for [Kubernetes](http://kubernetes.io). It prov
...
 
-```sbt
-libraryDependencies += "io.skuber" % "skuber_2.11" % "1.7.1"
-```
+  This example lists pods in `kube-system` namespace. It unsafely assumes that your kubeconfig file exists and kubernetes cluster is accessible:
```
Comment (doriordan) [inline README.md]: Can we remove the word `unsafely` here? For a quickstart I think just outlining the assumptions/prerequisites is good enough.

File: README.md (hunk 1 above)
Comment (doriordan) [reply README.md]: Also I'd prefer this to be in its own section before the quickstart, placing it between the ammonite and sbt parts of the quickstart could be a bit confusing,

File: README.md [hunk 3]
```diff
@@ -7,125 +11,131 @@ Skuber is a Scala client library for [Kubernetes](http://kubernetes.io). It prov
...
+  import akka.stream.ActorMaterializer
+  import scala.util.{Success, Failure}
 
-If you have a Skuber client using release v1.x and want to move to the strategic version 2 release, then check out the [migration guide](docs/MIGRATION_1-to-2.md).
+  implicit val system = ActorSystem()
+  implicit val materializer = ActorMaterializer()
+  implicit val dispatcher = system.dispatcher
 
-## Building
+  val cfg: Configuration = Configuration.parseKubeconfigFile().get
```
Comment (doriordan) [inline README.md]: To keep it as simple as possible I'd prefer to use default configuration
`  val k8s = k8sInit()`

File: README.md (hunk 1 above)
Comment (ngortheone) [reply README.md]: ack

File: README.md (hunk 2 above)
Comment (ngortheone) [reply README.md]: ack

**Comment by ngortheone** [comment]
@doriordan I have addressed all the issues that you pointed out :shipit:

**Comment by doriordan** [comment]
Thanks @ngortheone LGTM

**Pull request merged**